cmd/update/update.go

```diff
@@ -0,0 +1,63 @@
+package update
+
+//This update command updates to the latest kubescape release.
+//Example:-
+//          kubescape update
+
+import (
+	"fmt"
+	"os/exec"
+	"runtime"
+
+	logger "github.com/kubescape/go-logger"
+	"github.com/kubescape/kubescape/v2/core/cautils"
+	"github.com/spf13/cobra"
+)
+
+func GetUpdateCmd() *cobra.Command {
+	updateCmd := &cobra.Command{
+		Use:   "update",
+		Short: "Update your version",
+		Long:  ``,
+		RunE: func(_ *cobra.Command, args []string) error {
+			//Checking the user's version of kubescape to the latest release
+			if cautils.BuildNumber == cautils.LatestReleaseVersion {
+				//your version == latest version
+				fmt.Println("You are in the latest version")
+			} else {
+				//execute the install.sh if linux, install.ps1 for windows,.....depending on your OS
+				const OSTYPE string = runtime.GOOS
+				if OSTYPE == "linux" {
+					cautils.StartSpinner()
+					//run the installation command for linux
+					_, err := exec.Command("./install.sh").Output()
+					if err != nil {
+						logger.L().Fatal(err.Error())
+					}
+					cautils.StopSpinner()
+
+				} else if OSTYPE == "windows" {
+					cautils.StartSpinner()
+					//run the installation command for windows
+					_, err := exec.Command("./install.ps1").Output()
+					if err != nil {
+						logger.L().Fatal(err.Error())
+					}
+					cautils.StopSpinner()
+
+				} else {
+					cautils.StartSpinner()
+					//run the installation command for macOS
+					_, err := exec.Command("./kubescape-cli.rb").Output()
```

Comment:
Can you please explain why are you calling this file? Why not download using [homebrew tap](https://github.com/kubescape/homebrew-tap) or `install.sh`?

Reply:
I thought install.sh is only for linux, so went through the [homebrew-tap ](https://github.com/kubescape/homebrew-tap)repo and added the ```kubescape-cli.rb``` file to kubescape and linked it here .